tests/dryrun_test.go

```diff
@@ -235,7 +235,7 @@ var _ = Describe("[sig-compute]Dry-Run requests", decorators.SigCompute, func()
 			vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
```

Comment:
Are you referring to the `NewRandomMigration` to `NewMigration`?
If yes, Done :)
Otherwise, I agree that typically is a good idea to add a description! Here the things seemed to me quite self explanatory.
Please let me know, thanks

Reply:
yes, I meant
```
refactor: rename NewRandomMigration to NewMigration

Remove the `Random` part in the name function, since
there's nothing random. The only thing that is (almost)
random is the name, that is generated, but this pattern
is used in almost every resource in the testsuite (see vmi).

```